internal/lookout/ui/src/services/JobsLocalStorageService.ts

```diff
@@ -9,36 +9,44 @@ export type JobsLocalStorageState = {
   annotationColumns?: ColumnSpec<string>[]
 }
 
-function convertToLocalStorageState(loadedData: Record<string, unknown>): JobsLocalStorageState {
+function isColumnArray(data: unknown): boolean {
+  return data !== undefined && Array.isArray(data) && data.every((col: any) => isColumnSpec(col))
+}
+
+function loadedDataIsValid(loadedData: Record<string, unknown>): boolean {
+  return (
+    loadedData.autoRefresh !== undefined &&
+    typeof loadedData.autoRefresh == "boolean" &&
+    isColumnArray(loadedData.defaultColumns) &&
```

Comment:
Are defaultColumns and annotationColumns always defined?

Reply:
No, the null check happens in `isColumnArray`